### PR TITLE
Removes nearly all medical alt titles

### PR DESCRIPTION
**Pull request by Alberyk**
This pr removes nearly all medical alt titles, except the psychologist one.

Why: those alt titles are pretty much confusing and pointless, it also just causes people to create pages and pages of discussions trying to point out what each alt title can and can not do. They are not worth keeping.

Also, reduced the Physician age to match nurse characters' age.

### CODE REVIEW

File: code/game/jobs/job/medical.dm
Comment (NonQueueingMatt) [inline code/game/jobs/job/medical.dm:62]: This is odd. The wiki rightfully points out that one can only have a medical degree at 28 years of age. Shouldn't that still be the case?

**Review by NonQueueingMatt — COMMENTED** [review]
You will need to remove the trauma physician clothing and sprites as well. 

There is also this PR: https://github.com/Aurorastation/Aurora.3/pull/9577 which is superceded I think.

**Review by Arrow768 — APPROVED** [review]

**Comment by Arrow768** [comment]
> You will need to remove the trauma physician clothing and sprites as well.

Keep the clothing, I´m gonna reuse it for admin roles

**Review by Chada-1 — APPROVED** [review]

**Comment by sadkermit** [comment]
I’m not against the aim of this pull request and do believe that the numerous alt-roles are a problem and should be removed (though I disagree with your implementation of the solution [I'd ask for a feedback thread as others have expressed concerns about this PR, but I'm unsure if it would be my place to do so and it may also be a tad too late]), but do you not think this undermines another PR (#9577) which is already working on tackling the issues of medical alt-roles, one which also went further by re-naming the role such that they had less real life connotations? This PR was the result of a suggestion thread where a fair number of people discussed and agreed upon a set of homogenised roles and possible new names (granted it has been stale as I remember Skull mentioning he was busy and some names still had a bit of contention surrounding them). I think it would be quite brazen to not give any consideration and to disregard the views of those who discussed what they did and did not want from any changes to the Medical roles, especially when a nearly clear consensus was reached and a PR was already opened to implement these ideas.

**Comment by Alberyk** [comment]
That pr does far more than just removing alt titles, it changes all names. This pr just removes the alt titles but keeps the names. The other pr can be merged if it is ever completed.

**Comment by Arrow768** [comment]
Gonna merge this as #9577 can always be implemented after this has been merged.

**Pull request merged**
